spec/models/video_spec.rb

```diff
@@ -2,7 +2,44 @@
 
 describe Video do
 
-  it { should have_many(:categories)}
+  it {should have_many(:categories)}
   it {should validate_presence_of(:title)}
   it {should validate_presence_of(:description)}
-end
\ No newline at end of file
+  
+  describe "search_by_title" do
```

Comment:
In RSpec, usually its common to describe class methods with prefixed `.` and instance methods with `#`. Read more here => http://www.betterspecs.org/#describe